### DISCUSSION TITLE
What is server exception?

### DISCUSSION THREAD

**Discussion by mina1134** [UPPAALModelChecker/UPPAAL-Meta#222] [Q&A]
When I use random trace at simulation, I got the error repeatably called "ServerException" at specific function. Is it problem on my function? Usually If my function has any ploblem, I can see the red underline in declarations. But this error could not find part of the error.
![UPPALL_ERROR](https://github.com/UPPAALModelChecker/UPPAAL-Meta/assets/104436496/b99bbf16-6e19-4ce2-850a-e7a7d6befd04)

**Comment by mikucionisaau** [comment]
All the simulation, analysis checking and so is run by Uppaal engine in a separate "server" process. In this case, it looks like simulator operation ran into some kind of error, i.e. it is a runtime error caught during simulation as opposed to syntax error caught and highlighted in the editor.

Perhaps you can resize/increase the popup window to reveal the full error message.
The size of this popup message is a separate bug.

**Reply by mina1134** [reply]
Thanks to you, I found a way to check the full error sentence.   
And I got c++ library error "Error in function boost::math::arcsine_distribution<double>::arcsine_distribution: x_max argument is 6, but must be > x_min = 6!"  
Can I get a reason why x_max and x_min are assiged the same value?

**Reply by mikucionisaau** [reply]
Can you show us the piece of code which calls `random_arcsine` function?
It takes two arguments and it seems that in your case both of them are equal to `6`.
